### PR TITLE
[Data Mapper] Individual function icons

### PR DESCRIPTION
**Pull request by refortie**
image

AB#17356025

### CODE REVIEW

**Comment by hartra344** [comment]
Not related to this PR in particular but there's likely going to be some a11y issues iwth the black text/image on the dark colored circle. we may need to address that.

File: apps/data-mapper-standalone/src/components/DevApiTester.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const DevApiTester = () => {
 
   const getSchemaTree = async () => {
     try {
+      setApiResponse('Getting schema');
```
Comment (NicL9923) [inline apps/data-mapper-standalone/src/components/DevApiTester.tsx:55]: Ultra-nit-suggestion: 'Getting schema...' ?

**Review by NicL9923 — APPROVED** [review]
Awesome!

**Comment by refortie** [comment]
> Not related to this PR in particular but there's likely going to be some a11y issues iwth the black text/image on the dark colored circle. we may need to address that.

Yeah, I need to address the colors in another PR, they are technically oppositely themed right now, but I wanted to keep this focused on just the images for now

**Pull request merged**